trailmap/parse_input.py

```diff
@@ -0,0 +1,125 @@
+import os
+import sys
+import subprocess
+import json
+import xml.etree.ElementTree as ET
+import cyclus
+
+
+def parse_input(input, commodity_dictionary):
+    '''Builds dicionary of facilities and their incommodies and outcommodities.
+
+    inputs:
+        - input: a Cyclus XML input file
+        - commodity_dictionary: a dictionary
+
```

Comment:
I would remove that blank line here. (and all the others in the docs strings.)
r at least make it consistent with `get_facility_and_commod_names` methods